### DISCUSSION TITLE
kairship/workload/yaml

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#2374] [Announcements]
# kairship/workload/yaml

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/kairship/workload/yaml.html

**Comment by yt-huang** [comment]
部署策略的版本有问题，新版通过yaml通过多deployment负载
placement:
  clusterAffinities:
  - affinityName: default
    clusternames:
    - bztest001
      olm-cluster-test

通过老版本创建的多云管理负载会导致无法正常显示指定集群以及无法进行差异化部署
placement:
    clusterAffinity:
      clusterNames:
        - demo-stage
        - demo-dev

**Comment by yt-huang** [comment]
部署策略实例有误， clusterAffinities 下面是一个数组

**Comment by windsonsea** [comment]
Fixed in https://github.com/DaoCloud/DaoCloud-docs/pull/4583
